README.md

```diff
@@ -14,395 +14,245 @@
+# Train the model
+engine.fit(datamodule=datamodule, model=model)
 ```
 
-Alternatively, a model name could also be provided as an argument, where the scripts automatically finds the corresponding config file.
+</details>
+
+<details>
+<summary>Training via CLI</summary>
 
 ```bash
-python tools/train.py --model padim
+# To get help about the training arguments, run:
+anomalib train -h
+
+# To train the model on the default values, run:
+anomalib train --model anomalib.models.patchcore --data anomalib.data.MVTec
 ```
 
-where the currently available models are:
+</details>
 
-- [CFA](src/anomalib/models/cfa)
-- [CFlow](src/anomalib/models/cflow)
-- [DFKDE](src/anomalib/models/dfkde)
-- [DFM](src/anomalib/models/dfm)
-- [DRAEM](src/anomalib/models/draem)
-- [EfficientAd](src/anomalib/models/efficient_ad)
-- [FastFlow](src/anomalib/models/fastflow)
-- [GANomaly](src/anomalib/models/ganomaly)
-- [PADIM](src/anomalib/models/padim)
-- [PatchCore](src/anomalib/models/patchcore)
-- [Reverse Distillation](src/anomalib/models/reverse_distillation)
-- [STFPM](src/anomalib/models/stfpm)
+# 🤖 Inference
 
-## Feature extraction & (pre-trained) backbones
+Anomalib includes multiple inferencing scripts, including Torch, Lightning, Gradio, and OpenVINO inferencers to perform inference using the trained/exported model. Here we show an inference example using the Lightning inferencer. For other inferencers, please refer to the [Inference Documentation](https://anomalib.readthedocs.io).
 
-The pre-trained backbones come from [PyTorch Image Models (timm)](https://github.com/rwightman/pytorch-image-models), which are wrapped by `FeatureExtractor`.
+<details>
+<summary>Inference via API</summary>
+
+The following example demonstrates how to perform Lightning inference by loading a model from a checkpoint file.
+
+```python
+# Assuming the datamodule, model and engine is initialized from the previous step,
+# a prediction via a checkpoint file can be performed as follows:
+predictions = engine.predict(
+    datamodule=datamodule,
+    model=model,
+    ckpt_path="path/to/checkpoint.ckpt",
+)
+```
 
-For more information, please check our documentation or the [section about feature extraction in "Getting Started with PyTorch Image Models (timm): A Practitioner’s Guide"](https://towardsdatascience.com/getting-started-with-pytorch-image-models-timm-a-practitioners-guide-4e77b4bf9055#b83b:~:text=ready%20to%20train!-,Feature%20Extraction,-timm%20models%20also>).
+</details>
 
-Tips:
+<details>
+<summary>Inference via CLI</summary>
 
-- Papers With Code has an interface to easily browse models available in timm: [https://paperswithcode.com/lib/timm](https://paperswithcode.com/lib/timm)
+```bash
+# To get help about the arguments, run:
+anomalib predict -h
 
-- You can also find them with the function `timm.list_models("resnet*", pretrained=True)`
+# Predict via providing the model name
+anomalib predict --model anomalib.models.Patchcore
```

Comment:
This does not work. It complains about `--data` arg missing. I also think we should provide a `--ckpt_path` arg here, because `anomalib predict` is only useful when we specify a previously trained model.

Reply:
@ashwinvaidya17 ?